zetta_utils/__init__.py

```diff
@@ -3,12 +3,16 @@
 import os
 import sys
 import warnings
+import multiprocessing
 
 from .log import get_logger
 
 # Set global multiprocessing threshold
 MULTIPROCESSING_NUM_TASKS_THRESHOLD = 128
 
+# Unnecessary after 3.14 - not encapsulated in main due to workers
+multiprocessing.set_start_method("forkserver", force=True)
```

Comment:
Not an expert but all of the best practice guides I've come across recommend to never put this unguarded.